vda5050_connector/vda5050_connector_py/vda5050_controller.py

```diff
@@ -1208,19 +1208,18 @@ def _reject_order(self, order: VDAOrder, error: OrderRejectErrors, description:
         if error == OrderRejectErrors.ORDER_UPDATE_ERROR:
             # On orderUpdateError send orderUpdateId and orderId as reference
 
-            # TODO: Question: camelCase or snakeCase (order_id or orderId)
             error_references.append(
-                VDAErrorReference(reference_key="order_id", reference_value=order.order_id)
+                VDAErrorReference(reference_key="orderId", reference_value=order.order_id)
             )
             error_references.append(
                 VDAErrorReference(
-                    reference_key="order_update_id", reference_value=str(order.order_update_id)
+                    reference_key="orderUpdateId", reference_value=str(order.order_update_id)
                 )
             )
         elif error == OrderRejectErrors.NO_ROUTE_ERROR:
             # On noRouteError send 1st node as reference
             error_references.append(
-                VDAErrorReference(reference_key="node_id", reference_value=order.nodes[0].node_id)
+                VDAErrorReference(reference_key="nodeId", reference_value=order.nodes[0].node_id)
```

Comment:
This are ok to change because it seems they aren't handled by `convert_ros_message_to_json()`